src/odemis/util/linalg.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
Please write here, at the top of the file the copyright. So move it away from the "Notes", and check what's the actual license of scipy's code (and whether it can be relicensed as GPL-2, as the rest of Odemis).

Reply:
@pieleric [scipy has a BSD license](https://github.com/scipy/scipy/blob/master/LICENSE.txt) , do you know if that means it can be relicenseed as GPL-2?

Reply:
This 3-clause BSD license is compatible with GPL-2 (cf https://en.wikipedia.org/wiki/BSD_licenses). We need to make sure the copyright is properly indicated, at least in the header of this file.